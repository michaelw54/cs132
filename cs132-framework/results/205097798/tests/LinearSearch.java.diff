1,4c1,15
< ParseException: Encountered " <IDENTIFIER> "java "" at line 208, column 1.
< Was expecting:
<     "return" ...
<     
---
> 10
> 11
> 12
> 13
> 14
> 15
> 16
> 17
> 18
> 9999
> 0
> 1
> 1
> 0
> 55
