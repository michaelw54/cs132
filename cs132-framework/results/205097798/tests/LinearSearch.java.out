ParseException: Encountered " <IDENTIFIER> "java "" at line 208, column 1.
Was expecting:
    "return" ...
    
